python_ta/transforms/type_inference_visitor.py

```diff
@@ -217,6 +217,32 @@ def visit_assign(self, node: astroid.Assign) -> None:
             if isinstance(type_result, TypeFail):
                 node.inf_type = type_result
 
+    def visit_augassign(self, node: astroid.AugAssign) -> None:
+        if isinstance(node.value, astroid.Name):
```

Comment:
I'm not sure why this is a special case; seems like it's okay to use the `.inf_type` here as long as `self._assign_type` is called below?